data/tooltips.txt

```diff
@@ -68,6 +68,9 @@ tip "cooling inefficiency:"
 tip "cost:"
 	`Cost, in credits. If you have sold any used (depreciated) items today, you can buy them back for the same price you sold them for.`
 
+tip "disruption protection:"
+	`Protection against weapons that have a disruption effect. A value of 1 cuts the effect in half, and 2 cuts it to a third of what it would otherwise be.`
```

Comment:
Should these tips say `... total value of 1...`?

Reply:
It's the same language as the cargo and outfit scan power, so if these get changes then those should be changed too.